pages/_app.page.tsx

```diff
@@ -57,6 +59,17 @@ interface MyAppProps extends AppProps {
 export default function MyApp(props: MyAppProps) {
   const { Component, emotionCache = clientSideEmotionCache, pageProps } = props;
   const getLayout = Component.getLayout ?? ((page: React.ReactNode) => page);
+  const initializeMixpanel = useRootStore((store) => store.initializeMixpanel);
+
+  const MIXPANEL_TOKEN = process.env.NEXT_PUBLIC_MIXPANEL;
```

Comment:
There are 3 ways to set this variable:
1.) Hardcode into the .env file in the repo (easily abused)
2.) Add the token as a repo secret and add a step to the Github Actions pipeline to inject it into the build (less easily abused, but the token is in the build source code so it can still be extracted through "View Page Source")
3.) Code the token into a backend (Aave API), and re-write the PR to send events to an API endpoint restricted to app.aave.com origin, instead of sending to mixpanel directly on the UI

Reply:
agree, i think 3 is the best option

Reply:
I think it's ok if we leave it in the '.env' file. The token is only used to identify the project in mixpanel for events, not authentication

https://developer.mixpanel.com/reference/project-token

Reply:
If the token is public (in .env or in the client build), anyone can spoof any event i.e. you could send an event that you supplied 100000000 ETH